operations/mimir-mixin-compiled/dashboards/mimir-reads.json

```diff
@@ -924,7 +924,7 @@
                   "steppedLine": false,
                   "targets": [
                      {
-                        "expr": "# Query metrics before and after migration to new memcached backend.\nsum (\n  rate(cortex_cache_request_duration_seconds_count{name=~\"frontend.+\", cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read.*))\"}[$__rate_interval])\n  or\n  rate(thanos_memcached_operation_duration_seconds_count{name=\"frontend-cache\", cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read.*))\"}[$__rate_interval])\n)\n",
+                        "expr": "sum (\n  rate(thanos_memcached_operations_total{name=\"frontend-cache\", cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read.*))\"}[$__rate_interval])\n  or\n  rate(thanos_cache_operations_total{name=\"frontend-cache\", cluster=~\"$cluster\", job=~\"($namespace)/((query-frontend.*|cortex|mimir|mimir-read.*))\"}[$__rate_interval])\n)\n",
```

Comment:
@56quarters I just realized this query double counts because the labels are different, and so the `or` operator doesn't select "one or the other". I think we need a bit more transformation in our query to avoid double counting.

Example:
<img width="1695" alt="Screenshot 2023-02-17 at 09 20 27" src="https://user-images.githubusercontent.com/1701904/219591179-719afc93-0e77-475f-b0b0-75dbe9d758f5.png">

I haven't checked all other queries. Same issue may be in other places too (but not a real issue when we compute average or latency percentile since double tracking shouldn't change the math).

Reply:
Looks like the new `backend` label is causing the duplicates. I'll try adding `or ignoring(backend)` and see if the values line up.

Reply:
Confirmed that `or ignoring(backend)` fixes the double-counting for request rate but doesn't affect latency or hit ratio panels (like you said).